judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ProblemStatementCard.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { Divider } from '@blueprintjs/core';
+import { Item, ItemType } from 'modules/api/sandalphon/problemBundle';
+import { ProblemItemStatementCard } from './ProblemItemStatementCard/ProblemItemStatementCard';
+import { ProblemItemMultipleChoiceCard } from './ProblemItemMultipleChoiceCard/ProblemItemMultipleChoiceCard';
+
+import './ProblemStatementCard.css';
+import { ItemSubmission } from 'modules/api/sandalphon/submissionBundle';
+
+export interface ProblemStatementCardProps {
+  title: string;
+  alias: string;
+  items: Item[];
+  onItemAnswered: (itemJid: string, answer?: string) => any;
+  latestSubmission: { [id: string]: ItemSubmission };
+}
+
+export class ProblemStatementCard extends React.Component<ProblemStatementCardProps> {
+  generateOnAnswer = (itemJid: string) => {
+    return (choice?: string) => {
+      this.props.onItemAnswered(itemJid, choice);
+    };
+  };
+  render() {
+    const { title, alias, items, latestSubmission } = this.props;
+    return (
+      <React.Fragment>
```

Comment:
```suggestion
      <>
```

The "fish" syntax is supported.